### PR TITLE
Do not skip the the_geom_webmercator field

### LINKED ISSUE #2816: public tables header have more columns that values
screen shot 2015-03-18 at 10 34 59 am

I guess is because we don't request the_geom_webmercator

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
This fixes #2816. We were showing the `the_geom_webmercator` column, but we were skipping that field when retrieving the data, and that was causing the inconsistence:

image

Not skipping that field fixes the problem:

captura de pantalla 2015-03-24 a las 12 51 58

Just for the record, when we render the table, (at least) two queries are made: [a first query that retrieves the schema](https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb/models/tabledata.js#L739-L749) and then another query that fetches the data. The first query doesn't take into account any `skipfields` option, and that's the reason why we were seeing all the fields in the header. If we want to remove that field in the future we will have to modify [the way the sql to fetch the schema is generated](https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb/models/tabledata.js#L700-L703).

This change didn't break any test :cry:.

@viddo @xavijam can you please take a look? thx

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2498/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2499/))

File: lib/assets/javascripts/cartodb/public_table/table_public.js [hunk 1]
```diff
@@ -41,7 +41,6 @@
       this.columns = this.table.data();
       this.sqlView = new cdb.admin.SQLViewData();
       this.sqlView.syncMethod = 'read';
-      this.sqlView.options.attributes.skipfields = 'the_geom_webmercator';
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/public_table/table_public.js]: oh, skipfields attribute, I missed you. cc @javisantana

**Comment by xavijam** [comment]
For me :es:

**Comment by saleiva** [comment]
:tada:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2517/))

**Pull request merged**